frontend/src/scenes/notebooks/Nodes/NotebookNodeTimestampItem.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+    Node,
+    NodeViewContent,
+    NodeViewProps,
+    NodeViewWrapper,
+    ReactNodeViewRenderer,
+    mergeAttributes,
+} from '@tiptap/react'
+import { dayjs } from 'lib/dayjs'
+import { useValues } from 'kea'
+import {
+    sessionRecordingPlayerLogic,
+    SessionRecordingPlayerLogicProps,
+} from 'scenes/session-recordings/player/sessionRecordingPlayerLogic'
+import { NotebookNodeType } from '~/types'
+import { hasContent } from '../Notebook/utils'
+import clsx from 'clsx'
+
+const Component = (props: NodeViewProps): JSX.Element => {
+    const playbackTime = props.node.attrs.playbackTime
+    const sessionRecordingId = props.node.attrs.sessionRecordingId
+
+    const recordingLogicProps: SessionRecordingPlayerLogicProps = {
+        sessionRecordingId,
+        playerKey: `notebook-${sessionRecordingId}`,
+    }
+
+    const { currentPlayerTime } = useValues(sessionRecordingPlayerLogic(recordingLogicProps))
+
+    const isEmpty = !hasContent(props.node)
+
+    if (!isEmpty && !playbackTime) {
+        setTimeout(() => {
```

Comment:
I was seeing the same error @pauldambra faced in https://github.com/PostHog/posthog/pull/16415#issuecomment-1631346398. Adding a very short timeout worked but I feel like there has to be a better way here